spec/api-remote-spec.js

```diff
@@ -189,83 +188,83 @@ describe('remote module', () => {
       const buf = Buffer.from('test')
       const call = remote.require(path.join(fixtures, 'module', 'call.js'))
       const result = call.call(remote.createFunctionWithReturnValue(buf))
-      assert.strictEqual(result.constructor.name, 'Buffer')
+      expect(result.constructor.name).to.equal('Buffer')
     })
   })
 
   describe('remote modules', () => {
     it('includes browser process modules as properties', () => {
-      assert.strictEqual(typeof remote.app.getPath, 'function')
-      assert.strictEqual(typeof remote.webContents.getFocusedWebContents, 'function')
-      assert.strictEqual(typeof remote.clipboard.readText, 'function')
+      expect(remote.app.getPath).to.be.a('function')
+      expect(remote.webContents.getFocusedWebContents).to.be.a('function')
+      expect(remote.clipboard.readText).to.be.a('function')
     })
 
     it('returns toString() of original function via toString()', () => {
       const { readText } = remote.clipboard
-      assert(readText.toString().startsWith('function'))
+      expect(readText.toString().startsWith('function')).to.be.true()
 
       const { functionWithToStringProperty } = remote.require(path.join(fixtures, 'module', 'to-string-non-function.js'))
-      assert.strictEqual(functionWithToStringProperty.toString, 'hello')
+      expect(functionWithToStringProperty.toString).to.equal('hello')
     })
   })
 
   describe('remote object in renderer', () => {
     it('can change its properties', () => {
       const property = remote.require(path.join(fixtures, 'module', 'property.js'))
-      assert.strictEqual(property.property, 1127)
+      expect(property.property).to.equal(1127)
```

Comment:
these are better as: `expect(property).to.have.a.property('property').that.is.equal(1127)` etc